### PR TITLE
feat(play): custom namespace

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

When we debug locally, we can synchronously verify whether there are any problems with the custom namespace.

### CODE REVIEW

**Review by FrontEndDog — APPROVED** [review]
LGTM

File: play/app.example.vue [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-    <el-icon color="#409EFC">
-      <el-icon-check />
-    </el-icon>
-    <el-icon>
-      <el-icon-menu />
-    </el-icon>
-    <el-button>test</el-button>
-    <div v-loading="true" />
-  </div>
+  <el-config-provider namespace="ep">
```
Comment (tolking) [inline play/app.example.vue:2]: If the user does not config the `namespace="ep"`, the component style will be wrong.

Comment (btea) [reply play/app.example.vue:2]: Yes, so I gave a default configuration. And if the user modifies the configuration in the playground, the style modification can be synchronized by himself. At the same time, we can check whether the custom namespace is normal when testing the functions of other components.

Comment (tolking) [reply play/app.example.vue:2]: 如果默认启用了命名空间。如果用户从文档或者其它地方复制代码，这些代码可能不会有 `<el-config-provider namespace="ep">` ，这会引起样式错误。我们无法知道用户会在 `app.vue` 文件中输入什么代码。

或许可以默认禁用，注释说明如何启用命名空间

Comment (btea) [reply play/app.example.vue:2]: 这样也行。那现在把·`<el-config-provider namespace="ep">` 和 css 配置代码保留，但直接注释并添加说明。

**Review by tolking — APPROVED** [review]

**Pull request merged**